src/frame/worker/UserCrypto.ts

```diff
@@ -4,6 +4,25 @@ import SDKError from "../../lib/SDKError";
 import * as AES from "./crypto/aes";
 import loadRecrypt from "./crypto/recrypt";
 
+/**
+ * Requests augmentation on users current private key. The augmented private key is then encrypted and using the password derived key.
+ * @param {Uint8Array} privateKey
+ * @param {Uint8Array} augmentationFactor
+ */
+export function augmentPrivateKey(
+    passcode: string,
+    userPrivateKey: Uint8Array
```

Comment:
Given my comment above, I'm pretty sure what we get here is actually the users encrypted private key. We need to use the password they give us to derive their private key escrow key and decrypt the key.

Reply:
done